src/System.Data.SqlClient/tests/FunctionalTests/System.Data.SqlClient.Tests.builds

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@weshaggard Do we need to go update https://github.com/dotnet/corefx/blob/master/src/tests.builds#L16 to be something like `*/tests/**/*.builds`?